clang/lib/Lex/PPMacroExpansion.cpp

```diff
@@ -254,7 +285,7 @@ void Preprocessor::updateModuleMacroInfo(const IdentifierInfo *II,
 }
 
 void Preprocessor::dumpMacroInfo(const IdentifierInfo *II) {
-  ArrayRef<ModuleMacro*> Leaf;
+  ArrayRef<ModuleMacro *> Leaf;
```

Comment:
This patch has a bunch of unrelated formatting changes, so I'm not sure if anything else in this file actually changes.